ps-client.js

```diff
@@ -89,13 +90,48 @@
     this.timeout = timeout;
   };
 
+  PredictiveServiceClient.prototype.setSchema = function(cb) { 
+
+    var callback = function(error, resp) {
+      if (resp == null) {
+        cb(error);
+      } else {
+        if (resp.statusCode != 200) {
+          throw new Error("Error connecting to Dato Predictive Service %s." % this.end_point);
+        } else {
+          if (typeof(resp.data) != "string" && 'schema_version' in resp.data) {
```

Comment:
Should we check the following way instead:

if (typeof(resp.data) === "object") {...}